pkg/util/k8sutil/k8sutil.go

```diff
@@ -81,6 +81,40 @@ const (
 
 var ErrDiscoveryTokenNotProvided = errors.New("cluster token not provided, you must provide a token when clustering mode is discovery")
 
+var CreateSvc CreateService = func(ctx context.Context, kubecli kubernetes.Interface, svcName string, clusterName string, ns string, ports []v1.ServicePort, owner metav1.OwnerReference, publishNotReadyAddresses bool, service *api.ServicePolicy, annotations map[string]string) error {
```

Comment:
Do you need to assign this function in a variable here? Can't we just declare the function and pass it as a parameter in the CreateClientService call?

Reply:
I thought about setting the type to make sure that in the future we don't disrupt the tests, that's why I created the type in [line 284](https://github.com/topfreegames/etcd-operator/blob/85d350588290d1383364e23ac38c6875faeeedac/pkg/util/k8sutil/k8sutil.go#L284)